app/(normalLayout)/definition/page.tsx

```diff
@@ -167,6 +167,51 @@ export default async function Page() {
                   </ul>
                 </div>
               </div>
+              <div className="fr-col-12 fr-col-md-8">
+                <div className="block block--yellow block--fill">
+                  <h3 className="block__title">
+                    Signalement un erreur sur un bâtiment
```

Comment:
Oh lala mais j'ai écris n'importe quoi.
```suggestion
                    Signaler une erreur sur un bâtiment
```